Not as root incompatible with docker-based custompios build

### Is your feature request related to a problem? Please describe

When using CustomPiOs to build a custom image with crowsnest as a module, the build fails because of the following lines:

    if [[ "${SUDO_USER}" = "root" ]]; then
        not_as_root_msg
        exit 1
    fi

### Describe the solution you'd like

Allow root to install, generate a warning message instead and let it proceed.  Or add a new make target that disables this check, like make install-docker

### Describe alternatives you've considered

_No response_

### Additional information

_No response_

from my CustomPiOS build log:

Launch crowsnest install routine ...
sudo: unable to resolve host 8491fdd2a6cb: Name or service not known

        Please do NOT run this script as root!

        Login in as regular user and run with 'sudo make install'


make: *** [Makefile:17: install] Error 1
++++ echo_red 'build failed, unmounting image...'
++++ echo -e -n '\e[91m'
++++ echo build failed, unmounting image...
build failed, unmounting image...
++++ echo -e -n '\e[0m'


--------------

${SUDO_USER} evaluates to root when building from docker, so in the install.sh it triggers the not_as_root_msg, even if i call make install as non-root, i.e, sudo -u SOMEOTHERSUDOER make install.

can you clarify what you mean that the custompios module skips checking for root? the custompios module calls make install, which calls install.sh that has the check for root?

Here my test setup to get the users.
I created and executed a new module called `test` with an empty config and following code inside:
```
#!/usr/bin/env bash

# Error handling
set -Ee

# shellcheck disable=SC1091
source /common.sh

echo_green "==================================="
echo "echo \${USER}" > /test.sh
echo "echo \${BASE_USER}" >> /test.sh
echo "echo \${SUDO_USER}" >> /test.sh
chmod +x /test.sh
/test.sh
sudo /test.sh
echo_green "==================================="
```
This will print out to green "lines" and between the results. I then tested the same with a test branch of crowsnest, printed out there the different users and the results were the same.

--- a/tools/install.sh
+++ b/tools/install.sh
@@ -42,7 +42,7 @@ main() {
         exit 1
     fi
 
-    if [[ -z "${SUDO_USER}" ]]; then
+    if [[ -z "${SUDO_USER}" ]] && [[ "${CROWSNEST_UNATTENDED}" != "1" ]]; then
         need_sudo_msg
         exit 1
     fi

